crates/uv/src/commands/venv.rs

```diff
@@ -189,22 +188,23 @@ async fn venv_impl(
         }
     }
 
-    match &platform.os() {
-        Os::Windows => {
-            writeln!(
-                printer,
-                "Activate with .\\{}\\Scripts\\activate.ps1",
-                path.normalized_display().cyan()
-            )
-        },
-        _ => {
-            writeln!(
-                printer,
-                "Activate with source {}/bin/activate",
-                path.normalized_display().cyan()
-            )
-        }
-    }.into_diagnostic()?;
+    if cfg!(windows) {
+        writeln!(
+            printer,
+            "Activate with:\n\
+            - Powershell: .\\{0}\\Scripts\\activate.ps1\n\
+            - CMD: .\\{0}\\Scripts\\activate.bat",
```

Comment:
I usually don't write `.\` when I'm in CMD environment.

Reply:
removed `.\` from the prompt